src/resource-archive/index.ts

```diff
@@ -136,6 +136,12 @@ class Watcher {
       return;
     }
 
+    // There's no reponse body for us to archive on 304s
+    if (responseStatusCode === 304) {
+      await this.clientSend(request, 'Fetch.continueRequest', { requestId });
+      return;
+    }
```

Comment:
We could probably stick this in the same `if` as the non-GET-method one above. Not a dealbreaker though.

Reply:
It checks a different thing so I like it being separate for now, but we should definitely do another pass and tidy up these checks.